graylog2-server/src/test/java/org/graylog2/rest/resources/users/UsersResourceTest.java

```diff
@@ -132,6 +145,78 @@ public void savePreferencesSuccess() throws ValidationException {
         verify(userManagementService, times(1)).save(isA(UserImpl.class));
     }
 
+    @Test
+    public void createTokenForInternalUserSucceeds() {
+        final Map<String, Object> userProps = Map.of(UserImpl.USERNAME, USERNAME);
+        final String tokenName = "tokenName";
+        final String token = "someToken";
+        final DateTime lastAccess = Tools.nowUTC();
+        final Map<String, Object> tokenProps = Map.of(AccessTokenImpl.NAME, tokenName, AccessTokenImpl.TOKEN, token, AccessTokenImpl.LAST_ACCESS, lastAccess);
+        final ObjectId tokenId = new ObjectId();
+        final AccessToken accessToken = new AccessTokenImpl(tokenId, tokenProps);
+        final Token expected = Token.create(tokenId.toHexString(), tokenName, token, lastAccess);
+
+        when(userManagementService.loadById(eq(USERNAME))).thenReturn(userImplFactory.create(userProps));
+        when(subject.isPermitted(eq(USERS_TOKENCREATE + ":" + USERNAME))).thenReturn(true);
```

Comment:
Remove unnecessary `eq()`:
```suggestion
        when(userManagementService.loadById(USERNAME)).thenReturn(userImplFactory.create(userProps));
        when(subject.isPermitted(USERS_TOKENCREATE + ":" + USERNAME)).thenReturn(true);
```

Do the same for other instances in this file.